app/src/main/java/com/pr0gramm/app/services/UserService.kt

```diff
@@ -351,6 +351,13 @@ class UserService(private val api: Api,
         return Graph(start.millis.toDouble(), now.millis.toDouble(), points)
     }
 
+    fun loadBenis(userId:Int = loginState.id):Observable<BenisRecord>{
+        val date = Instant(0)
```

Comment:
Nope, aktuell wird eine blockierende Anfrage ausgeführt und dann das Ergebnis in ein Observable gepackt. Wenn du die Methode aufrufst, blockiert die immer noch. Das hast du vermutlich nicht bemerkt, weil du sie immer noch per Hand aus einem anderen Thread aufrufst. 
Später kannst du sie direkt auf dem main thread aufrufen, ohne dass es ein Problem ist.